docs/commands/split.md

```diff
@@ -1,33 +1,41 @@
 # `split`
 
-The `split` command takes an API definition file and creates a multi-file structure out of it by extracting referenced parts into standalone, separate files. Essentially, the `split` command does the opposite of the `bundle` command.
+## Introduction
 
+The `split` command takes an API definition file and creates a [multi-file structure](../index.md#multi-file-approach) out of it by extracting referenced parts into standalone, separate files. Essentially, the `split` command does the opposite of the `bundle` command.
 
-<div class="warning">
+:::warning
 The `split` command doesn't support OpenAPI 2.0 definitions.
-</div>
+:::
 
+## Usage
 
-### `split` usage
-
-
-```shell
-Positionals:
-entrypoint                                                            [string]
-
-Options:
---version  Show version number.                                      [boolean]
---help     Show help.                                                [boolean]
---outDir   Output directory where files will be saved      [string] [required]
+```bash
+openapi split <entrypoint> --outDir=<path>
+openapi split [--help] [--version]
+openapi split --version
 ```
 
+## Options
 
-The command:
+Option                    | Type      | Required     | Default     | Description
+--------------------------|:---------:|:------------:|:-----------:|------------
+`entrypoint`              | `string`  | yes          | -           | Path to the API definition filename that you want to calculate statistics for
```

Comment:
issue: The description text seems to be copied from the `stats` command?

Reply:
Yes, sorry for that :(
Corrected